
# HG changeset patch
# User Robert Sayre <sayrer@gmail.com>
# Date 1280559674 25200
# Node ID bb4a143ee814097076640c37df84ef7f1303b63b
# Parent  9e08308ca8cb9d4584ff990f441f363f3d81bd57
Fix ShaderLang.h bustage. Commas at the end of enum lists.

diff --git a/gfx/angle/include/GLSLANG/ShaderLang.h b/gfx/angle/include/GLSLANG/ShaderLang.h
--- a/gfx/angle/include/GLSLANG/ShaderLang.h
+++ b/gfx/angle/include/GLSLANG/ShaderLang.h
@@ -47,7 +47,7 @@ ANGLE_API int ShFinalize();
 typedef enum {
     EShLangVertex,
     EShLangFragment,
-    EShLangCount,
+    EShLangCount
 } EShLanguage;
 
 //
@@ -56,7 +56,7 @@ typedef enum {
 //
 typedef enum {
     EShSpecGLES2,
-    EShSpecWebGL,
+    EShSpecWebGL
 } EShSpec;
 
 //
@@ -66,12 +66,12 @@ typedef enum {
     EShOptNoGeneration,
     EShOptNone,
     EShOptSimple,       // Optimizations that can be done quickly
-    EShOptFull,         // Optimizations that will take more time
+    EShOptFull          // Optimizations that will take more time
 } EShOptimizationLevel;
 
 enum TDebugOptions {
     EDebugOpNone               = 0x000,
-    EDebugOpIntermediate       = 0x001,  // Writes intermediate tree into info-log.
+    EDebugOpIntermediate       = 0x001   // Writes intermediate tree into info-log.
 };
 
 //

